tests/monotouch-test/HealthKit/HKHealthStoreTest.cs

```diff
@@ -13,15 +13,25 @@
 
 using Foundation;
 using HealthKit;
-using UIKit;
 using NUnit.Framework;
+#if MONOMAC
+using AppKit;
+#else
+using UIKit;
+#endif
 
 namespace MonoTouchFixtures.HealthKit {
 
 	[TestFixture]
 	[Preserve (AllMembers = true)]
 	public class HKHealthStoreTest {
 
+		[SetUp]
+		public void SetUp ()
+		{
+			TestRuntime.AssertXcodeVersion (14, 0);
```

Comment:
I think this should be okay since these 'BiologicalSex' and 'BloodType' are supported in:
iOS 8.0+
iPadOS 8.0+
macOS 13.0+ Beta
Mac Catalyst 13.0+
watchOS 2.0+

So as long as Mac is not being tested in anything before Xcode 14, this should be okay I think?

Reply:
Oh wait, are you saying that we only need the check for mac, and then it will run other platforms against earlier xcode versions and that is okay?

Reply:
Yes, that's exactly what I mean :smile: